topics/metagenomics/tutorials/general-tutorial/tutorial.md

```diff
@@ -455,23 +506,24 @@ the two samples?
 {: .hands_on}
 
 
-To further explore the community structure, we can visualize it with dedicated tools such as Phinch:
+To further explore the community structure, we can visualize it with dedicated tools such as Phinch.
 
 > ### {% icon hands_on %} Hands-on: Visualization of the community structure with Phinch
 >
 > 1. **Make.biom** {% icon tool %} with the following parameters
->   - "shared" to Make.shared
->   - "constaxonomy" to taxonomy output from the first run of `classify.otu` (collection)
+>   - "shared" to `Make.shared` output
+>   - "constaxonomy" to taxonomy output from the first run of `Classify.otu` (collection)
+> 1. On "calculator" collection, click on "Visualize" and then "Phinch"
```

Comment:
this is the way to start the IE right? replace with "expand the dataset and click on the "view biom at phinch" link" ?